src/api/GraphQuery/types.ts

```diff
@@ -223,13 +215,11 @@ export interface IGetQueriesParams extends IDataSourceParams {
   type: GraphQueryType;
 }
 
-export interface ICreateQueryParams extends IDataSourceParams {
+export interface ICreateQueryParams extends IDataSourceParams, SharingOptions {
```

Comment:
@yigitakbulut I'm not sure we understood each other. For me it should not extend SharingOptions, it should have a property `sharingOptions` of type `SharingOptions`